packages/app-store/routing-forms/components/react-awesome-query-builder/config/BasicConfig.ts

```diff
@@ -30,11 +27,19 @@ const conjunctions: Conjunctions = {
     label: "And",
     jsonLogicConj: "and",
     reversedConj: "OR",
+    formatConj: () => "",
+    sqlFormatConj: () => "",
+    spelFormatConj: () => "",
+    mongoConj: "",
```

Comment:
raqb methods throw TS error for these. So, let it have them even though we don't need them